docs/release-notes/latest.md

```diff
@@ -72,10 +72,11 @@ tags:
 ### Added
 
 <!-- Dependency graphs? -->
-- [Dependency Paths](/semgrep-supply-chain/dependency-search#view-the-dependency-path) are now available for the following languages and package managers:
-  - **JavaScript**: all package managers are supported by Semgrep.
+- [Dependency Paths](/semgrep-supply-chain/dependency-search#view-the-dependency-path) are now available in **private beta** for the following languages and package managers:
```

Comment:
I think this version (private beta) is the correct one as it requires an FF. Either way they should all be aligned.